src/store/postgres/PgDatastore.ts

```diff
@@ -358,8 +366,8 @@ export class PgDataStore implements IStore {
                 room_id: matrixId,
                 user_id: (remoteData as IRemoteUserAdminData).matrixUser,
             };
-            statement = PgDataStore.BuildUpsertStatement("admin_rooms", "(user_id)", Object.keys(adminProps));
-            await this.pgPool.query(statement, Object.values(adminProps));
+            // We don't upsert here.
```

Comment:
Why not? :)

The way it is now seems like we may end up creating duplicate admin rooms by mistake. Perhaps we could give them a constraint in the schema?

Reply:
We do constrain the room_id to be unique, but not the user_id. I think it's probably a followup task to ensure the admin room is unique for each user.

Reply:
https://github.com/matrix-org/matrix-bifrost/issues/277